lib/mongoid/changeable.rb

```diff
@@ -291,18 +318,21 @@ def create_dirty_default_change_check(name, meth)
         end
       end
 
-      # Creates the dirty change previous value accessor.
+      # Creates the dirty change previous value accessors.
       #
       # @example Create the accessor.
-      #   Model.create_dirty_previous_value_accessor("name", "alias")
+      #   Model.create_dirty_previous_value_accessors("name", "alias")
       #
       # @param [ String ] name The attribute name.
       # @param [ String ] meth The name of the accessor.
-      def create_dirty_previous_value_accessor(name, meth)
+      def create_dirty_previous_value_accessors(name, meth)
```

Comment:
If this is a private API it should be marked with `@api private`, otherwise I don't think it can be changed like this.

Reply:
This method is in a private section. I believe this assumes that such API is private.
On the other hand, there is no need to rename this method, to be honest. I'll revert the change.